src/llrt-function.ts

```diff
@@ -123,6 +130,10 @@ export class LlrtFunction extends NodejsFunction {
           afterBundling: (i, o) => [
             // Download llrt binary from GitHub release and cache it
             `if [ ! -e ${posix.join(i, cacheDir, 'bootstrap')} ]; then
+              if [ -n "${props.llrtCustomBinaryDirectory}" ]; then
+                echo "Error: Could not find bootstrap binary in ${props.llrtCustomBinaryDirectory}"
+                exit 1
+              fi
               mkdir -p ${posix.join(i, cacheDir)}
               cd ${posix.join(i, cacheDir)}
               curl -L -o llrt_temp.zip ${binaryUrl}
```

Comment:
Let's skip the download part entirely when `props.llrtBinaryPath` is configured.

Reply:
Yeah, I think renaming it and changing the functionality is the best solution here